pyo3-ffi/src/cpython/frameobject.rs

```diff
@@ -62,13 +62,22 @@ opaque_struct!(PyFrameObject);
 #[cfg_attr(windows, link(name = "pythonXY"))]
 extern "C" {
     pub static mut PyFrame_Type: PyTypeObject;
+
+    #[cfg(Py_3_13)]
+    pub static mut PyFrameLocalsProxy_Type: PyTypeObject;
```

Comment:
It looks like as of `3.13` this symbol and the other symbol you've added is in a file named `cpython/pyframe.h`. Lots of other stuff that's currently in PyO3's `cpython/frameobject.rs` also got moved there.

Can you go through `cpython/frameobject.rs` and `cpython/pyframe.rs` to make sure that as of 3.13 the contents of these files reflect what's in the upstream headers?

The goal with `pyo3-ffi` is to make the FFI definitions match what's in the CPython headers as closely as possible as of the most recent version of Python we support. Of course we don't catch everything every time we bump support for a new Python version, so it's an ever-losing battle to keep these up-to-date.